
Property changes on: .
___________________________________________________________________
Modified: svn:ignore
   - nb-configuration.xml
target
nbproject
.pom.xml.swp
target(2)

   + nb-configuration.xml
target
nbproject
.pom.xml.swp
target(2)
.classpath
.project
.settings


Index: src/main/java/jsyntaxpane/SyntaxTester.form
===================================================================
--- src/main/java/jsyntaxpane/SyntaxTester.form	(revision 136)
+++ src/main/java/jsyntaxpane/SyntaxTester.form	(working copy)
@@ -3,17 +3,19 @@
 <Form version="1.5" maxVersion="1.6" type="org.netbeans.modules.form.forminfo.JFrameFormInfo">
   <Properties>
     <Property name="defaultCloseOperation" type="int" value="3"/>
-    <Property name="title" type="java.lang.String" value="JSyntaxPane Tester"/>
+    <Property name="title" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+      <ResourceString bundle="jsyntaxpane/Bundle.properties" key="SyntaxTester.title" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+    </Property>
   </Properties>
   <SyntheticProperties>
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
   </SyntheticProperties>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
@@ -61,7 +63,9 @@
     <Component class="javax.swing.JLabel" name="lblCaretPos">
       <Properties>
         <Property name="horizontalAlignment" type="int" value="11"/>
-        <Property name="text" type="java.lang.String" value="Caret Position"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="SyntaxTester.lblCaretPos.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
     </Component>
     <Container class="javax.swing.JScrollPane" name="jScrollPane1">
@@ -73,7 +77,9 @@
       <SubComponents>
         <Component class="javax.swing.JEditorPane" name="jEdtTest">
           <Properties>
-            <Property name="contentType" type="java.lang.String" value=""/>
+            <Property name="contentType" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+              <ResourceString bundle="jsyntaxpane/Bundle.properties" key="SyntaxTester.jEdtTest.contentType" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+            </Property>
             <Property name="font" type="java.awt.Font" editor="org.netbeans.beaninfo.editors.FontEditor">
               <Font name="Monospaced" size="13" style="0"/>
             </Property>
@@ -92,7 +98,9 @@
         <Property name="font" type="java.awt.Font" editor="org.netbeans.beaninfo.editors.FontEditor">
           <Font name="Courier New" size="12" style="0"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Token under cursor"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="SyntaxTester.lblToken.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
     </Component>
     <Component class="javax.swing.JComboBox" name="jCmbLangs">
Index: src/main/java/jsyntaxpane/SyntaxTester.java
===================================================================
--- src/main/java/jsyntaxpane/SyntaxTester.java	(revision 136)
+++ src/main/java/jsyntaxpane/SyntaxTester.java	(working copy)
@@ -54,13 +54,14 @@
         jToolBar1 = new javax.swing.JToolBar();
 
         setDefaultCloseOperation(javax.swing.WindowConstants.EXIT_ON_CLOSE);
-        setTitle("JSyntaxPane Tester");
+        java.util.ResourceBundle bundle = java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle"); // NOI18N
+        setTitle(bundle.getString("SyntaxTester.title")); // NOI18N
 
         lblCaretPos.setHorizontalAlignment(javax.swing.SwingConstants.TRAILING);
-        lblCaretPos.setText("Caret Position");
+        lblCaretPos.setText(bundle.getString("SyntaxTester.lblCaretPos.text")); // NOI18N
 
-        jEdtTest.setContentType("");
-        jEdtTest.setFont(new java.awt.Font("Monospaced", 0, 13)); // NOI18N
+        jEdtTest.setContentType(bundle.getString("SyntaxTester.jEdtTest.contentType")); // NOI18N
+        jEdtTest.setFont(new java.awt.Font("Monospaced", 0, 13));
         jEdtTest.setCaretColor(new java.awt.Color(153, 204, 255));
         jEdtTest.addCaretListener(new javax.swing.event.CaretListener() {
             public void caretUpdate(javax.swing.event.CaretEvent evt) {
@@ -70,7 +71,7 @@
         jScrollPane1.setViewportView(jEdtTest);
 
         lblToken.setFont(new java.awt.Font("Courier New", 0, 12));
-        lblToken.setText("Token under cursor");
+        lblToken.setText(bundle.getString("SyntaxTester.lblToken.text")); // NOI18N
 
         jCmbLangs.setMaximumRowCount(20);
         jCmbLangs.setFocusable(false);
@@ -130,7 +131,7 @@
 				lblToken.setText(t.toString() + ": " + tData);
 			} else {
 				// null token, remove the status
-				lblToken.setText("NO Token at cursor");
+				lblToken.setText(java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("NO_TOKEN_AT_CURSOR"));
 			}
 		}
 
Index: src/main/java/jsyntaxpane/actions/DocumentSearchData.java
===================================================================
--- src/main/java/jsyntaxpane/actions/DocumentSearchData.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/DocumentSearchData.java	(working copy)
@@ -14,6 +14,7 @@
 package jsyntaxpane.actions;
 
 import java.awt.Component;
+import java.text.MessageFormat;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.regex.Matcher;
@@ -240,8 +241,8 @@
 	 */
 	public void msgNotFound(Component target) {
 		JOptionPane.showMessageDialog(SwingUtilities.getWindowAncestor(target),
-			"Search String " + getPattern() + " not found",
-			"Find", JOptionPane.INFORMATION_MESSAGE);
+			MessageFormat.format(java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("DocumentSearchData.SearchStringNotFound"), getPattern()),
+			java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("DocumentSearchData.Find"), JOptionPane.INFORMATION_MESSAGE);
 	}
 
 	/**
Index: src/main/java/jsyntaxpane/actions/ScriptAction.java
===================================================================
--- src/main/java/jsyntaxpane/actions/ScriptAction.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/ScriptAction.java	(working copy)
@@ -59,8 +59,8 @@
 				showScriptError(target, ex);
 			}
 		} else {
-			JOptionPane.showMessageDialog(target, "Action does not have script function configured",
-				"Error in Script", JOptionPane.WARNING_MESSAGE);
+			JOptionPane.showMessageDialog(target, java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptAction.NoScriptConfigured"),
+				java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptAction.ErrorInScript"), JOptionPane.WARNING_MESSAGE);
 		}
 	}
 
@@ -91,14 +91,14 @@
 				showScriptError(null, ex);
 			}
 		} else {
-			JOptionPane.showMessageDialog(null, "No script is found in: " + url,
-				"Error in Script", JOptionPane.WARNING_MESSAGE);
+			JOptionPane.showMessageDialog(null, java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptAction.NoScriptFoundIn") + url,
+				java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptAction.ErrorInScript"), JOptionPane.WARNING_MESSAGE);
 		}
 	}
 
 	private void showScriptError(JTextComponent target, Exception ex) {
 		JOptionPane.showMessageDialog(target, ex.getMessage(),
-			"Error in Script", JOptionPane.WARNING_MESSAGE);
+			java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptAction.ErrorInScript"), JOptionPane.WARNING_MESSAGE);
 	}
 	/**
 	 * The key used to store the Script Name for the this action
Index: src/main/java/jsyntaxpane/actions/ScriptRunnerAction.java
===================================================================
--- src/main/java/jsyntaxpane/actions/ScriptRunnerAction.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/ScriptRunnerAction.java	(working copy)
@@ -14,6 +14,8 @@
 package jsyntaxpane.actions;
 
 import java.awt.event.ActionEvent;
+import java.text.MessageFormat;
+
 import javax.script.ScriptEngine;
 import javax.script.ScriptEngineManager;
 import javax.script.ScriptException;
@@ -46,8 +48,8 @@
             }
         } catch (ScriptException ex) {
             JOptionPane.showMessageDialog(SwingUtilities.getWindowAncestor(target),
-                    "Error executing script:\n" + ex.getMessage(),
-                    "Script Error",
+                    java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptRunnerAction.ErrorExecutingScript") + ex.getMessage(),
+                    java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptRunnerAction.ScriptError"),
                     JOptionPane.ERROR_MESSAGE);
             ActionUtils.setCaretPosition(target,
                     ex.getLineNumber(),
@@ -64,7 +66,7 @@
         }
         if (engine == null) {
             int result = JOptionPane.showOptionDialog(target,
-                    "Script Engine for [" + scriptExtension + "] not found. Disable this Action?",
+                    MessageFormat.format(java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ScriptRunnerAction.ScriptEngineNotFound"), scriptExtension),
                     "jsyntaxpane",
                     JOptionPane.YES_NO_OPTION,
                     JOptionPane.ERROR_MESSAGE,
Index: src/main/java/jsyntaxpane/actions/ShowAbbsAction.java
===================================================================
--- src/main/java/jsyntaxpane/actions/ShowAbbsAction.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/ShowAbbsAction.java	(working copy)
@@ -43,7 +43,7 @@
 			Map<String, String> abbs = kit.getAbbreviations();
 			if (abbs == null || abbs.isEmpty()) {
 				JOptionPane.showMessageDialog(target,
-					"No Abbreviations exist for this content type");
+					java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("ShowAbbsAction.NoAbbsForType"));
 			} else {
 				new ShowAbbsDialog((JEditorPane) target, abbs);
 			}
Index: src/main/java/jsyntaxpane/actions/gui/ComboCompletionDialog.form
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/ComboCompletionDialog.form	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/ComboCompletionDialog.form	(working copy)
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.1" encoding="UTF-8" ?>
 
-<Form version="1.3" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
+<Form version="1.5" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
   <Properties>
     <Property name="defaultCloseOperation" type="int" value="2"/>
     <Property name="resizable" type="boolean" value="false"/>
@@ -10,11 +10,11 @@
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
   </SyntheticProperties>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
Index: src/main/java/jsyntaxpane/actions/gui/GotoLineDialog.form
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/GotoLineDialog.form	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/GotoLineDialog.form	(working copy)
@@ -1,8 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.1" encoding="UTF-8" ?>
 
-<Form version="1.3" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
+<Form version="1.5" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
   <Properties>
-    <Property name="title" type="java.lang.String" value="Goto Line"/>
+    <Property name="title" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+      <ResourceString bundle="jsyntaxpane/Bundle.properties" key="GotoLineDialog.title" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+    </Property>
     <Property name="modal" type="boolean" value="true"/>
     <Property name="name" type="java.lang.String" value="" noResource="true"/>
     <Property name="resizable" type="boolean" value="false"/>
@@ -11,11 +13,11 @@
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
   </SyntheticProperties>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
@@ -64,7 +66,9 @@
         <Property name="action" type="javax.swing.Action" editor="org.netbeans.modules.form.RADConnectionPropertyEditor">
           <Connection component="jCmbLineNumbers" name="action" type="property"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Go"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="GotoLineDialog.jBtnOk.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
       <Events>
         <EventHandler event="actionPerformed" listener="java.awt.event.ActionListener" parameters="java.awt.event.ActionEvent" handler="jBtnOkActionPerformed"/>
Index: src/main/java/jsyntaxpane/actions/gui/GotoLineDialog.java
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/GotoLineDialog.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/GotoLineDialog.java	(working copy)
@@ -55,7 +55,8 @@
         jCmbLineNumbers = new javax.swing.JComboBox();
         jBtnOk = new javax.swing.JButton();
 
-        setTitle("Goto Line");
+        java.util.ResourceBundle bundle = java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle"); // NOI18N
+        setTitle(bundle.getString("GotoLineDialog.title")); // NOI18N
         setModal(true);
         setName(""); // NOI18N
         setResizable(false);
@@ -68,7 +69,7 @@
         });
 
         jBtnOk.setAction(jCmbLineNumbers.getAction());
-        jBtnOk.setText("Go");
+        jBtnOk.setText(bundle.getString("GotoLineDialog.jBtnOk.text")); // NOI18N
         jBtnOk.addActionListener(new java.awt.event.ActionListener() {
             public void actionPerformed(java.awt.event.ActionEvent evt) {
                 jBtnOkActionPerformed(evt);
Index: src/main/java/jsyntaxpane/actions/gui/HTMLPreviewFrame.form
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/HTMLPreviewFrame.form	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/HTMLPreviewFrame.form	(working copy)
@@ -1,9 +1,11 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.1" encoding="UTF-8" ?>
 
-<Form version="1.3" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JFrameFormInfo">
+<Form version="1.5" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JFrameFormInfo">
   <Properties>
     <Property name="defaultCloseOperation" type="int" value="2"/>
-    <Property name="title" type="java.lang.String" value="HTML Preview"/>
+    <Property name="title" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+      <ResourceString bundle="jsyntaxpane/Bundle.properties" key="HTMLPreviewFrame.title" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+    </Property>
   </Properties>
   <SyntheticProperties>
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
@@ -12,11 +14,11 @@
     <EventHandler event="windowClosed" listener="java.awt.event.WindowListener" parameters="java.awt.event.WindowEvent" handler="onWindowClosed"/>
   </Events>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
@@ -45,7 +47,9 @@
       <SubComponents>
         <Component class="javax.swing.JEditorPane" name="jEdtHtml">
           <Properties>
-            <Property name="contentType" type="java.lang.String" value="text/html"/>
+            <Property name="contentType" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+              <ResourceString bundle="jsyntaxpane/Bundle.properties" key="HTMLPreviewFrame.jEdtHtml.contentType" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+            </Property>
             <Property name="editable" type="boolean" value="false"/>
           </Properties>
         </Component>
Index: src/main/java/jsyntaxpane/actions/gui/HTMLPreviewFrame.java
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/HTMLPreviewFrame.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/HTMLPreviewFrame.java	(working copy)
@@ -61,14 +61,15 @@
         jEdtHtml = new javax.swing.JEditorPane();
 
         setDefaultCloseOperation(javax.swing.WindowConstants.DISPOSE_ON_CLOSE);
-        setTitle("HTML Preview");
+        java.util.ResourceBundle bundle = java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle"); // NOI18N
+        setTitle(bundle.getString("HTMLPreviewFrame.title")); // NOI18N
         addWindowListener(new java.awt.event.WindowAdapter() {
             public void windowClosed(java.awt.event.WindowEvent evt) {
                 onWindowClosed(evt);
             }
         });
 
-        jEdtHtml.setContentType("text/html");
+        jEdtHtml.setContentType(bundle.getString("HTMLPreviewFrame.jEdtHtml.contentType")); // NOI18N
         jEdtHtml.setEditable(false);
         jScrollPane1.setViewportView(jEdtHtml);
 
Index: src/main/java/jsyntaxpane/actions/gui/QuickFindDialog.form
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/QuickFindDialog.form	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/QuickFindDialog.form	(working copy)
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.1" encoding="UTF-8" ?>
 
 <Form version="1.5" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
   <Properties>
@@ -14,11 +14,11 @@
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
   </SyntheticProperties>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
@@ -58,7 +58,9 @@
             <Property name="labelFor" type="java.awt.Component" editor="org.netbeans.modules.form.ComponentChooserEditor">
               <ComponentRef name="jTxtFind"/>
             </Property>
-            <Property name="text" type="java.lang.String" value="Quick Find"/>
+            <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+              <ResourceString bundle="jsyntaxpane/Bundle.properties" key="QuickFindDialog.jLabel1.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+            </Property>
           </Properties>
         </Component>
         <Component class="javax.swing.JToolBar$Separator" name="jSeparator2">
@@ -115,7 +117,9 @@
         <Component class="javax.swing.JCheckBox" name="jChkIgnoreCase">
           <Properties>
             <Property name="mnemonic" type="int" value="67"/>
-            <Property name="text" type="java.lang.String" value="Ignore Case"/>
+            <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+              <ResourceString bundle="jsyntaxpane/Bundle.properties" key="QuickFindDialog.jChkIgnoreCase.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+            </Property>
             <Property name="focusable" type="boolean" value="false"/>
             <Property name="opaque" type="boolean" value="false"/>
             <Property name="verticalTextPosition" type="int" value="3"/>
@@ -127,7 +131,9 @@
         <Component class="javax.swing.JCheckBox" name="jChkRegExp">
           <Properties>
             <Property name="mnemonic" type="int" value="82"/>
-            <Property name="text" type="java.lang.String" value="Reg Exp"/>
+            <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+              <ResourceString bundle="jsyntaxpane/Bundle.properties" key="QuickFindDialog.jChkRegExp.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+            </Property>
             <Property name="focusable" type="boolean" value="false"/>
             <Property name="opaque" type="boolean" value="false"/>
             <Property name="verticalTextPosition" type="int" value="3"/>
@@ -139,7 +145,9 @@
         <Component class="javax.swing.JCheckBox" name="jChkWrap">
           <Properties>
             <Property name="mnemonic" type="int" value="87"/>
-            <Property name="text" type="java.lang.String" value="Wrap"/>
+            <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+              <ResourceString bundle="jsyntaxpane/Bundle.properties" key="QuickFindDialog.jChkWrap.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+            </Property>
             <Property name="focusable" type="boolean" value="false"/>
             <Property name="opaque" type="boolean" value="false"/>
             <Property name="verticalTextPosition" type="int" value="3"/>
Index: src/main/java/jsyntaxpane/actions/gui/QuickFindDialog.java
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/QuickFindDialog.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/QuickFindDialog.java	(working copy)
@@ -145,7 +145,8 @@
         jToolBar1.add(jSeparator1);
 
         jLabel1.setLabelFor(jTxtFind);
-        jLabel1.setText("Quick Find");
+        java.util.ResourceBundle bundle = java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle"); // NOI18N
+        jLabel1.setText(bundle.getString("QuickFindDialog.jLabel1.text")); // NOI18N
         jToolBar1.add(jLabel1);
         jToolBar1.add(jSeparator2);
 
@@ -182,7 +183,7 @@
         jToolBar1.add(jBtnNext);
 
         jChkIgnoreCase.setMnemonic('C');
-        jChkIgnoreCase.setText("Ignore Case");
+        jChkIgnoreCase.setText(bundle.getString("QuickFindDialog.jChkIgnoreCase.text")); // NOI18N
         jChkIgnoreCase.setFocusable(false);
         jChkIgnoreCase.setOpaque(false);
         jChkIgnoreCase.setVerticalTextPosition(javax.swing.SwingConstants.BOTTOM);
@@ -190,7 +191,7 @@
         jChkIgnoreCase.addActionListener(this);
 
         jChkRegExp.setMnemonic('R');
-        jChkRegExp.setText("Reg Exp");
+        jChkRegExp.setText(bundle.getString("QuickFindDialog.jChkRegExp.text")); // NOI18N
         jChkRegExp.setFocusable(false);
         jChkRegExp.setOpaque(false);
         jChkRegExp.setVerticalTextPosition(javax.swing.SwingConstants.BOTTOM);
@@ -198,7 +199,7 @@
         jChkRegExp.addActionListener(this);
 
         jChkWrap.setMnemonic('W');
-        jChkWrap.setText("Wrap");
+        jChkWrap.setText(bundle.getString("QuickFindDialog.jChkWrap.text")); // NOI18N
         jChkWrap.setFocusable(false);
         jChkWrap.setOpaque(false);
         jChkWrap.setVerticalTextPosition(javax.swing.SwingConstants.BOTTOM);
@@ -229,7 +230,7 @@
 		if (dsd.get().doFindNext(target.get())) {
 			jLblStatus.setText(null);
 		} else {
-			jLblStatus.setText("not found");
+			jLblStatus.setText(java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("QuickFindDialog.NotFound"));
 		}
 }//GEN-LAST:event_jBtnNextActionPerformed
 
@@ -237,7 +238,7 @@
 		if (dsd.get().doFindPrev(target.get())) {
 			jLblStatus.setText(null);
 		} else {
-			jLblStatus.setText("not found");
+			jLblStatus.setText(java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("QuickFindDialog.NotFound"));
 		}
 }//GEN-LAST:event_jBtnPrevActionPerformed
 
@@ -290,7 +291,7 @@
 			jLblStatus.setText(null);
 			t.setCaretPosition(oldCaretPosition);
 			if (!d.doFindNext(t)) {
-				jLblStatus.setText("Not found");
+				jLblStatus.setText(java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle").getString("QuickFindDialog.NotFound"));
 			} else {
 				jLblStatus.setText(null);
 			}
Index: src/main/java/jsyntaxpane/actions/gui/ReflectCompletionDialog.form
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/ReflectCompletionDialog.form	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/ReflectCompletionDialog.form	(working copy)
@@ -1,6 +1,6 @@
 <?xml version="1.1" encoding="UTF-8" ?>
 
-<Form version="1.3" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
+<Form version="1.5" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
   <Properties>
     <Property name="defaultCloseOperation" type="int" value="2"/>
     <Property name="name" type="java.lang.String" value="CompletionDialog" noResource="true"/>
@@ -11,11 +11,11 @@
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
   </SyntheticProperties>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
Index: src/main/java/jsyntaxpane/actions/gui/ReplaceDialog.form
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/ReplaceDialog.form	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/ReplaceDialog.form	(working copy)
@@ -1,8 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.1" encoding="UTF-8" ?>
 
-<Form version="1.3" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
+<Form version="1.5" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
   <Properties>
-    <Property name="title" type="java.lang.String" value="Find and Replace"/>
+    <Property name="title" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+      <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.title" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+    </Property>
     <Property name="name" type="java.lang.String" value="" noResource="true"/>
     <Property name="resizable" type="boolean" value="false"/>
   </Properties>
@@ -10,11 +12,11 @@
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
   </SyntheticProperties>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
@@ -32,14 +34,11 @@
               </Group>
               <EmptySpace max="-2" attributes="0"/>
               <Group type="103" groupAlignment="0" attributes="0">
-                  <Group type="102" alignment="0" attributes="0">
-                      <Component id="jChkRegex" min="-2" max="-2" attributes="0"/>
-                      <EmptySpace type="unrelated" max="-2" attributes="0"/>
-                      <Component id="jChkWrap" min="-2" pref="105" max="-2" attributes="0"/>
-                  </Group>
                   <Component id="jCmbFind" alignment="0" pref="289" max="32767" attributes="0"/>
                   <Component id="jCmbReplace" alignment="1" pref="289" max="32767" attributes="1"/>
-                  <Component id="jChkIgnoreCase" min="-2" max="-2" attributes="0"/>
+                  <Component id="jChkRegex" alignment="0" pref="289" max="32767" attributes="0"/>
+                  <Component id="jChkWrap" alignment="0" pref="289" max="32767" attributes="0"/>
+                  <Component id="jChkIgnoreCase" alignment="0" pref="289" max="32767" attributes="0"/>
               </Group>
               <EmptySpace max="-2" attributes="0"/>
               <Group type="103" groupAlignment="0" attributes="0">
@@ -47,7 +46,7 @@
                   <Component id="jBtnNext" alignment="0" pref="107" max="32767" attributes="1"/>
                   <Component id="jBtnPrev" alignment="0" pref="107" max="32767" attributes="1"/>
                   <Component id="jTglHighlight" alignment="1" pref="107" max="32767" attributes="0"/>
-                  <Component id="jBtnReplaceAll" alignment="1" max="32767" attributes="1"/>
+                  <Component id="jBtnReplaceAll" alignment="1" pref="107" max="32767" attributes="1"/>
               </Group>
               <EmptySpace max="-2" attributes="0"/>
           </Group>
@@ -69,10 +68,12 @@
                   <Component id="jLblReplace" alignment="3" min="-2" max="-2" attributes="0"/>
               </Group>
               <EmptySpace max="-2" attributes="0"/>
-              <Component id="jBtnReplace" min="-2" max="-2" attributes="0"/>
+              <Group type="103" groupAlignment="3" attributes="0">
+                  <Component id="jBtnReplace" alignment="3" min="-2" max="-2" attributes="0"/>
+                  <Component id="jChkWrap" alignment="3" min="-2" pref="23" max="-2" attributes="0"/>
+              </Group>
               <EmptySpace min="-2" pref="3" max="-2" attributes="0"/>
               <Group type="103" groupAlignment="3" attributes="0">
-                  <Component id="jChkWrap" alignment="3" min="-2" pref="23" max="-2" attributes="0"/>
                   <Component id="jChkRegex" alignment="3" min="-2" max="-2" attributes="0"/>
                   <Component id="jBtnReplaceAll" alignment="3" min="-2" max="-2" attributes="1"/>
               </Group>
@@ -93,7 +94,9 @@
         <Property name="labelFor" type="java.awt.Component" editor="org.netbeans.modules.form.ComponentChooserEditor">
           <ComponentRef name="jCmbFind"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Find"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jLblFind.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
     </Component>
     <Component class="javax.swing.JButton" name="jBtnNext">
@@ -102,7 +105,9 @@
           <Image iconType="3" name="/META-INF/images/small-icons/go-next.png"/>
         </Property>
         <Property name="mnemonic" type="int" value="78"/>
-        <Property name="text" type="java.lang.String" value="Next"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jBtnNext.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
       <Events>
         <EventHandler event="actionPerformed" listener="java.awt.event.ActionListener" parameters="java.awt.event.ActionEvent" handler="jBtnNextActionPerformed"/>
@@ -114,7 +119,9 @@
           <Image iconType="3" name="/META-INF/images/small-icons/go-previous.png"/>
         </Property>
         <Property name="mnemonic" type="int" value="78"/>
-        <Property name="text" type="java.lang.String" value="Previous"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jBtnPrev.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
       <Events>
         <EventHandler event="actionPerformed" listener="java.awt.event.ActionListener" parameters="java.awt.event.ActionEvent" handler="jBtnPrevActionPerformed"/>
@@ -126,7 +133,9 @@
           <Image iconType="3" name="/META-INF/images/small-icons/edit-find-replace-all.png"/>
         </Property>
         <Property name="mnemonic" type="int" value="72"/>
-        <Property name="text" type="java.lang.String" value="Replace All"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jBtnReplaceAll.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
       <Events>
         <EventHandler event="actionPerformed" listener="java.awt.event.ActionListener" parameters="java.awt.event.ActionEvent" handler="jBtnReplaceAllActionPerformed"/>
@@ -135,20 +144,28 @@
     <Component class="javax.swing.JCheckBox" name="jChkWrap">
       <Properties>
         <Property name="mnemonic" type="int" value="87"/>
-        <Property name="text" type="java.lang.String" value="Wrap around"/>
-        <Property name="toolTipText" type="java.lang.String" value="Wrap to beginning when end is reached"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jChkWrap.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
+        <Property name="toolTipText" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jChkWrap.toolTipText" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
     </Component>
     <Component class="javax.swing.JCheckBox" name="jChkRegex">
       <Properties>
         <Property name="mnemonic" type="int" value="82"/>
-        <Property name="text" type="java.lang.String" value="Regular Expression"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jChkRegex.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
     </Component>
     <Component class="javax.swing.JCheckBox" name="jChkIgnoreCase">
       <Properties>
         <Property name="mnemonic" type="int" value="73"/>
-        <Property name="text" type="java.lang.String" value="Ignore Case"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jChkIgnoreCase.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
     </Component>
     <Component class="javax.swing.JLabel" name="jLblReplace">
@@ -157,7 +174,9 @@
         <Property name="labelFor" type="java.awt.Component" editor="org.netbeans.modules.form.ComponentChooserEditor">
           <ComponentRef name="jCmbReplace"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Replace"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jLblReplace.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
     </Component>
     <Component class="javax.swing.JToggleButton" name="jTglHighlight">
@@ -165,7 +184,9 @@
         <Property name="icon" type="javax.swing.Icon" editor="org.netbeans.modules.form.editors2.IconEditor">
           <Image iconType="3" name="/META-INF/images/small-icons/highlight.png"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Highlight"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jTglHighlight.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
       <Events>
         <EventHandler event="actionPerformed" listener="java.awt.event.ActionListener" parameters="java.awt.event.ActionEvent" handler="jTglHighlightActionPerformed"/>
@@ -192,7 +213,9 @@
         <Property name="icon" type="javax.swing.Icon" editor="org.netbeans.modules.form.editors2.IconEditor">
           <Image iconType="3" name="/META-INF/images/small-icons/edit-find-replace.png"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Replace"/>
+        <Property name="text" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+          <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ReplaceDialog.jBtnReplace.text" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+        </Property>
       </Properties>
       <Events>
         <EventHandler event="actionPerformed" listener="java.awt.event.ActionListener" parameters="java.awt.event.ActionEvent" handler="jBtnReplaceActionPerformed"/>
Index: src/main/java/jsyntaxpane/actions/gui/ReplaceDialog.java
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/ReplaceDialog.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/ReplaceDialog.java	(working copy)
@@ -111,17 +111,18 @@
         jCmbFind = new javax.swing.JComboBox();
         jBtnReplace = new javax.swing.JButton();
 
-        setTitle("Find and Replace");
+        java.util.ResourceBundle bundle = java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle"); // NOI18N
+        setTitle(bundle.getString("ReplaceDialog.title")); // NOI18N
         setName(""); // NOI18N
         setResizable(false);
 
         jLblFind.setDisplayedMnemonic('F');
         jLblFind.setLabelFor(jCmbFind);
-        jLblFind.setText("Find");
+        jLblFind.setText(bundle.getString("ReplaceDialog.jLblFind.text")); // NOI18N
 
         jBtnNext.setIcon(new javax.swing.ImageIcon(getClass().getResource("/META-INF/images/small-icons/go-next.png"))); // NOI18N
         jBtnNext.setMnemonic('N');
-        jBtnNext.setText("Next");
+        jBtnNext.setText(bundle.getString("ReplaceDialog.jBtnNext.text")); // NOI18N
         jBtnNext.addActionListener(new java.awt.event.ActionListener() {
             public void actionPerformed(java.awt.event.ActionEvent evt) {
                 jBtnNextActionPerformed(evt);
@@ -130,7 +131,7 @@
 
         jBtnPrev.setIcon(new javax.swing.ImageIcon(getClass().getResource("/META-INF/images/small-icons/go-previous.png"))); // NOI18N
         jBtnPrev.setMnemonic('N');
-        jBtnPrev.setText("Previous");
+        jBtnPrev.setText(bundle.getString("ReplaceDialog.jBtnPrev.text")); // NOI18N
         jBtnPrev.addActionListener(new java.awt.event.ActionListener() {
             public void actionPerformed(java.awt.event.ActionEvent evt) {
                 jBtnPrevActionPerformed(evt);
@@ -139,7 +140,7 @@
 
         jBtnReplaceAll.setIcon(new javax.swing.ImageIcon(getClass().getResource("/META-INF/images/small-icons/edit-find-replace-all.png"))); // NOI18N
         jBtnReplaceAll.setMnemonic('H');
-        jBtnReplaceAll.setText("Replace All");
+        jBtnReplaceAll.setText(bundle.getString("ReplaceDialog.jBtnReplaceAll.text")); // NOI18N
         jBtnReplaceAll.addActionListener(new java.awt.event.ActionListener() {
             public void actionPerformed(java.awt.event.ActionEvent evt) {
                 jBtnReplaceAllActionPerformed(evt);
@@ -147,21 +148,21 @@
         });
 
         jChkWrap.setMnemonic('W');
-        jChkWrap.setText("Wrap around");
-        jChkWrap.setToolTipText("Wrap to beginning when end is reached");
+        jChkWrap.setText(bundle.getString("ReplaceDialog.jChkWrap.text")); // NOI18N
+        jChkWrap.setToolTipText(bundle.getString("ReplaceDialog.jChkWrap.toolTipText")); // NOI18N
 
         jChkRegex.setMnemonic('R');
-        jChkRegex.setText("Regular Expression");
+        jChkRegex.setText(bundle.getString("ReplaceDialog.jChkRegex.text")); // NOI18N
 
         jChkIgnoreCase.setMnemonic('I');
-        jChkIgnoreCase.setText("Ignore Case");
+        jChkIgnoreCase.setText(bundle.getString("ReplaceDialog.jChkIgnoreCase.text")); // NOI18N
 
         jLblReplace.setDisplayedMnemonic('R');
         jLblReplace.setLabelFor(jCmbReplace);
-        jLblReplace.setText("Replace");
+        jLblReplace.setText(bundle.getString("ReplaceDialog.jLblReplace.text")); // NOI18N
 
         jTglHighlight.setIcon(new javax.swing.ImageIcon(getClass().getResource("/META-INF/images/small-icons/highlight.png"))); // NOI18N
-        jTglHighlight.setText("Highlight");
+        jTglHighlight.setText(bundle.getString("ReplaceDialog.jTglHighlight.text")); // NOI18N
         jTglHighlight.addActionListener(new java.awt.event.ActionListener() {
             public void actionPerformed(java.awt.event.ActionEvent evt) {
                 jTglHighlightActionPerformed(evt);
@@ -173,7 +174,7 @@
         jCmbFind.setEditable(true);
 
         jBtnReplace.setIcon(new javax.swing.ImageIcon(getClass().getResource("/META-INF/images/small-icons/edit-find-replace.png"))); // NOI18N
-        jBtnReplace.setText("Replace");
+        jBtnReplace.setText(bundle.getString("ReplaceDialog.jBtnReplace.text")); // NOI18N
         jBtnReplace.addActionListener(new java.awt.event.ActionListener() {
             public void actionPerformed(java.awt.event.ActionEvent evt) {
                 jBtnReplaceActionPerformed(evt);
@@ -191,20 +192,18 @@
                     .addComponent(jLblReplace))
                 .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
                 .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
-                    .addGroup(layout.createSequentialGroup()
-                        .addComponent(jChkRegex)
-                        .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
-                        .addComponent(jChkWrap, javax.swing.GroupLayout.PREFERRED_SIZE, 105, javax.swing.GroupLayout.PREFERRED_SIZE))
                     .addComponent(jCmbFind, 0, 289, Short.MAX_VALUE)
                     .addComponent(jCmbReplace, javax.swing.GroupLayout.Alignment.TRAILING, 0, 289, Short.MAX_VALUE)
-                    .addComponent(jChkIgnoreCase))
+                    .addComponent(jChkRegex, javax.swing.GroupLayout.DEFAULT_SIZE, 289, Short.MAX_VALUE)
+                    .addComponent(jChkWrap, javax.swing.GroupLayout.DEFAULT_SIZE, 289, Short.MAX_VALUE)
+                    .addComponent(jChkIgnoreCase, javax.swing.GroupLayout.DEFAULT_SIZE, 289, Short.MAX_VALUE))
                 .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
                 .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
                     .addComponent(jBtnReplace, javax.swing.GroupLayout.Alignment.TRAILING, javax.swing.GroupLayout.DEFAULT_SIZE, 107, Short.MAX_VALUE)
                     .addComponent(jBtnNext, javax.swing.GroupLayout.DEFAULT_SIZE, 107, Short.MAX_VALUE)
                     .addComponent(jBtnPrev, javax.swing.GroupLayout.DEFAULT_SIZE, 107, Short.MAX_VALUE)
                     .addComponent(jTglHighlight, javax.swing.GroupLayout.Alignment.TRAILING, javax.swing.GroupLayout.DEFAULT_SIZE, 107, Short.MAX_VALUE)
-                    .addComponent(jBtnReplaceAll, javax.swing.GroupLayout.Alignment.TRAILING, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
+                    .addComponent(jBtnReplaceAll, javax.swing.GroupLayout.Alignment.TRAILING, javax.swing.GroupLayout.DEFAULT_SIZE, 107, Short.MAX_VALUE))
                 .addContainerGap())
         );
         layout.setVerticalGroup(
@@ -221,10 +220,11 @@
                     .addComponent(jCmbReplace, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)
                     .addComponent(jLblReplace))
                 .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
-                .addComponent(jBtnReplace)
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
+                    .addComponent(jBtnReplace)
+                    .addComponent(jChkWrap, javax.swing.GroupLayout.PREFERRED_SIZE, 23, javax.swing.GroupLayout.PREFERRED_SIZE))
                 .addGap(3, 3, 3)
                 .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
-                    .addComponent(jChkWrap, javax.swing.GroupLayout.PREFERRED_SIZE, 23, javax.swing.GroupLayout.PREFERRED_SIZE)
                     .addComponent(jChkRegex)
                     .addComponent(jBtnReplaceAll))
                 .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
Index: src/main/java/jsyntaxpane/actions/gui/ShowAbbsDialog.form
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/ShowAbbsDialog.form	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/ShowAbbsDialog.form	(working copy)
@@ -1,9 +1,11 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.1" encoding="UTF-8" ?>
 
-<Form version="1.3" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
+<Form version="1.5" maxVersion="1.7" type="org.netbeans.modules.form.forminfo.JDialogFormInfo">
   <Properties>
     <Property name="defaultCloseOperation" type="int" value="2"/>
-    <Property name="title" type="java.lang.String" value="Abbreviations"/>
+    <Property name="title" type="java.lang.String" editor="org.netbeans.modules.i18n.form.FormI18nStringEditor">
+      <ResourceString bundle="jsyntaxpane/Bundle.properties" key="ShowAbbsDialog.title" replaceFormat="java.util.ResourceBundle.getBundle(&quot;{bundleNameSlashes}&quot;).getString(&quot;{key}&quot;)"/>
+    </Property>
     <Property name="locationByPlatform" type="boolean" value="true"/>
     <Property name="minimumSize" type="java.awt.Dimension" editor="org.netbeans.beaninfo.editors.DimensionEditor">
       <Dimension value="[600, 300]"/>
@@ -15,11 +17,11 @@
     <SyntheticProperty name="formSizePolicy" type="int" value="1"/>
   </SyntheticProperties>
   <AuxValues>
-    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="0"/>
+    <AuxValue name="FormSettings_autoResourcing" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
     <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_generateMnemonicsCode" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="false"/>
+    <AuxValue name="FormSettings_i18nAutoMode" type="java.lang.Boolean" value="true"/>
     <AuxValue name="FormSettings_layoutCodeTarget" type="java.lang.Integer" value="1"/>
     <AuxValue name="FormSettings_listenerGenerationStyle" type="java.lang.Integer" value="0"/>
     <AuxValue name="FormSettings_variablesLocal" type="java.lang.Boolean" value="false"/>
Index: src/main/java/jsyntaxpane/actions/gui/ShowAbbsDialog.java
===================================================================
--- src/main/java/jsyntaxpane/actions/gui/ShowAbbsDialog.java	(revision 136)
+++ src/main/java/jsyntaxpane/actions/gui/ShowAbbsDialog.java	(working copy)
@@ -63,7 +63,8 @@
         jEdtAbbr = new javax.swing.JEditorPane();
 
         setDefaultCloseOperation(javax.swing.WindowConstants.DISPOSE_ON_CLOSE);
-        setTitle("Abbreviations");
+        java.util.ResourceBundle bundle = java.util.ResourceBundle.getBundle("jsyntaxpane/Bundle"); // NOI18N
+        setTitle(bundle.getString("ShowAbbsDialog.title")); // NOI18N
         setLocationByPlatform(true);
         setMinimumSize(new java.awt.Dimension(600, 300));
         setModal(true);
Index: src/main/resources/jsyntaxpane/Bundle.properties
===================================================================
--- src/main/resources/jsyntaxpane/Bundle.properties	(revision 0)
+++ src/main/resources/jsyntaxpane/Bundle.properties	(revision 0)
@@ -0,0 +1,44 @@
+# To change this template, choose Tools | Templates
+# and open the template in the editor.
+
+SyntaxTester.title=JSyntaxPane Tester
+SyntaxTester.lblCaretPos.text=Caret Position
+SyntaxTester.lblToken.text=Token under cursor
+SyntaxTester.jEdtTest.contentType=
+NO_TOKEN_AT_CURSOR=NO Token at cursor
+
+HTMLPreviewFrame.title=HTML Preview
+HTMLPreviewFrame.jEdtHtml.contentType=text/html
+QuickFindDialog.jChkWrap.text=Wrap
+QuickFindDialog.jChkRegExp.text=Reg Exp
+QuickFindDialog.jChkIgnoreCase.text=Ignore Case
+QuickFindDialog.jLabel1.text=Quick Find
+QuickFindDialog.NotFound=not found
+ReplaceDialog.jChkRegex.text=Regular Expression
+ReplaceDialog.jChkIgnoreCase.text=Ignore Case
+ReplaceDialog.jLblReplace.text=Replace
+ReplaceDialog.jTglHighlight.text=Highlight
+ReplaceDialog.jBtnReplaceAll.text=Replace All
+ReplaceDialog.jBtnPrev.text=Previous
+ReplaceDialog.jChkWrap.toolTipText=Wrap to beginning when end is reached
+ReplaceDialog.jChkWrap.text=Wrap around
+ReplaceDialog.title=Find and Replace
+ReplaceDialog.jBtnNext.text=Next
+ReplaceDialog.jLblFind.text=Find
+ReplaceDialog.jBtnReplace.text=Replace
+ShowAbbsDialog.title=Abbreviations
+# To change this template, choose Tools | Templates
+# and open the template in the editor.
+GotoLineDialog.title=Goto Line
+GotoLineDialog.jBtnOk.text=Go
+
+DocumentSearchData.SearchStringNotFound=Search String {0} not found
+DocumentSearchData.Find=Find
+ScriptAction.NoScriptConfigured=Action does not have script function configured
+ScriptAction.ErrorInScript=Error in Script
+ScriptAction.NoScriptFoundIn=No script is found in:
+ScriptRunnerAction.ErrorExecutingScript=Error executing script:\\n
+ScriptRunnerAction.ScriptError=Script Error
+ScriptRunnerAction.ScriptEngineNotFound=Script Engine for [{0}] not found. Disable this Action?
+ShowAbbsAction.NoAbbsForType=No Abbreviations exist for this content type
+

Property changes on: src\main\resources\jsyntaxpane\Bundle.properties
___________________________________________________________________
Added: svn:mime-type
   + text/plain
Added: svn:keywords
   + Date Revision
Added: svn:eol-style
   + native

Index: src/main/resources/jsyntaxpane/Bundle_de.properties
===================================================================
--- src/main/resources/jsyntaxpane/Bundle_de.properties	(revision 0)
+++ src/main/resources/jsyntaxpane/Bundle_de.properties	(revision 0)
@@ -0,0 +1,40 @@
+# To change this template, choose Tools | Templates
+# and open the template in the editor.
+
+SyntaxTester.title=JSyntaxPane Tester
+SyntaxTester.lblCaretPos.text=Caret-Position
+SyntaxTester.lblToken.text=Token unter dem Cursor
+SyntaxTester.jEdtTest.contentType=
+NO_TOKEN_AT_CURSOR=Kein Token am Cursor
+
+GotoLineDialog.title=Gehe zu Zeile
+GotoLineDialog.jBtnOk.text=Ok
+HTMLPreviewFrame.title=HTML-Vorschau
+HTMLPreviewFrame.jEdtHtml.contentType=text/html
+QuickFindDialog.jChkWrap.text=Ganzes Dokument
+QuickFindDialog.jChkRegExp.text=Regex
+QuickFindDialog.jChkIgnoreCase.text=Gro/Klein ignorieren
+QuickFindDialog.jLabel1.text=Schnellsuche
+QuickFindDialog.NotFound=nicht gefunden
+ReplaceDialog.jChkRegex.text=Regulrer Ausdruck
+ReplaceDialog.jChkIgnoreCase.text=Gro/Klein ignorieren
+ReplaceDialog.jLblReplace.text=Ersetzen
+ReplaceDialog.jTglHighlight.text=Hervorheben
+ReplaceDialog.jBtnReplaceAll.text=Alle Ersetzen
+ReplaceDialog.jBtnPrev.text=Zurck
+ReplaceDialog.jChkWrap.toolTipText=Von vorne weitersuchen wenn das Ende erreicht ist
+ReplaceDialog.jChkWrap.text=Ganzes Dokument
+ReplaceDialog.title=Suchen und Ersetzen
+ReplaceDialog.jBtnNext.text=Weiter
+ReplaceDialog.jLblFind.text=Suchen
+ReplaceDialog.jBtnReplace.text=Ersetzen
+
+DocumentSearchData.SearchStringNotFound=Suchbegriff {0} nicht gefunden
+DocumentSearchData.Find=Suchen
+ScriptAction.NoScriptConfigured=Aktion hat kein Skript zugeordnet
+ScriptAction.ErrorInScript=Fehler im Skript
+ScriptAction.NoScriptFoundIn=Kein Skript gefunden in:
+ScriptRunnerAction.ErrorExecutingScript=Fehler bei Skriptausfhrung:\\n
+ScriptRunnerAction.ScriptError=Skript-Fehler
+ScriptRunnerAction.ScriptEngineNotFound=Skript-Engine fr [{0}] nicht gefunden. Aktion deaktivieren?
+ShowAbbsAction.NoAbbsForType=Es gibt keine Abkrzungen fr diesen Content-Type

Property changes on: src\main\resources\jsyntaxpane\Bundle_de.properties
___________________________________________________________________
Added: svn:mime-type
   + text/plain
Added: svn:keywords
   + Date Revision
Added: svn:eol-style
   + native

